static/js/emoji_picker.js

```diff
@@ -633,6 +633,23 @@ exports.build_emoji_popover = function (elt, id) {
     register_popover_events(popover);
 };
 
+exports.is_only_emoji_message = function (message) {
```

Comment:
Can you write test for this function which tests various cases of emoji being present in a message?

Reply:
@amanagr, Sorry, if I am mistaken here. But, should I write test for this function which checks if a message passed should be classified within `large-emoji` or default emoji by something like: 
```
run_test("check_large_emoji", () => {
....
});
```
By default, there are markdown tests within `markdown_test_cases.json` and `marked`.